doc/releases/changelog-0.9.0.md

```diff
@@ -2,161 +2,175 @@
 
 <h3>New features</h3>
 
-* Experimental integration of the PennyLane capture module is available. It currently only supports
-  quantum gates, without control flow.
-  [(#1109)](https://github.com/PennyLaneAI/catalyst/pull/1109)
-
-  To trigger the PennyLane pipeline for capturing the program as a JaxPR, one needs to simply
-  set `experimental_capture=True` in the qjit decorator.
-
-  ```python
-  import pennylane as qml
-  from catalyst import qjit
-
-  dev = qml.device("lightning.qubit", wires=1)
+* Catalyst now supports the specification of shot-vectors when used with `qml.sample` measurements
+  on the `lightning.qubit` device.
+  [(#1051)](https://github.com/PennyLaneAI/catalyst/pull/1051)
 
-  @qjit(experimental_capture=True)
-  @qml.qnode(dev)
-  def circuit():
-      qml.Hadamard(0)
-      qml.CNOT([0, 1])
-      return qml.expval(qml.Z(0))
-  ```
+  Shot-vectors allow shots to be specified as a list of shots, `[20, 1, 100]`, or as a tuple of the
+  form `((num_shots, repetitions), ...)` such that `((20, 3), (1, 100))` is equivalent to
+  `shots=[20, 20, 20, 1, 1, ..., 1]`.
 
-* Shot-vector support for Catalyst: Introduces support for shot-vectors in Catalyst, currently
-  available for `qml.sample` measurements in the `lightning.qubit` device. Shot-vectors now allow
-  elements of the form `((20, 5),)`, which is equivalent to `(20,)*5` or `(20, 20, 20, 20, 20)`.
-  Furthermore, multiple `qml.sample` calls can now be returned from the same program, and can be
-  structured using Python containers. For example, a program can return a dictionary like
-  `return {"first": qml.sample(), "second": qml.sample()}`.
-  [(#1051)](https://github.com/PennyLaneAI/catalyst/pull/1051)
+  This can result in more efficient quantum execution, as a single job representing the total number
+  of shots is executed on the quantum device, with the measurement post-processing then
+  coarse-grained with respect to the shot-vector.
 
   For example,
 
   ```python
-  import pennylane as qml
-  from catalyst import qjit
-
   dev = qml.device("lightning.qubit", wires=1, shots=((5, 2), 7))
 
   @qjit
   @qml.qnode(dev)
   def circuit():
-    qml.Hadamard(0)
-    return qml.sample()
+      qml.Hadamard(0)
+      return qml.sample()
   ```
 
   ```pycon
   >>> circuit()
   (Array([[0], [1], [0], [1], [1]], dtype=int64),
   Array([[0], [1], [1], [0], [1]], dtype=int64),
-  Array([[1], [0], [1], [1], [0], [1],[0]], dtype=int64))
+  Array([[1], [0], [1], [1], [0], [1], [0]], dtype=int64))
```

Comment:
Did `make format` detect changelog or did you find this manually? Both impressive (though the latter is way more impressive)

Reply:
Credit to @josh146's eagle eyes on this one!